pipelines/_ingredients/LSSTCam-imSim/DRP.yaml

```diff
@@ -34,14 +37,8 @@ tasks:
       # to compute them for analyzeAmpOffsetMetadata to report on as metrics.
       doAmpOffset: true
       ampOffset.doApplyAmpOffset: false
-  calibrate:
-    class: lsst.pipe.tasks.calibrate.CalibrateTask
-    config:
-      connections.astromRefCat: "cal_ref_cat_2_2"
-      connections.photoRefCat: "cal_ref_cat_2_2"
-      python: |
-        config.astromRefObjLoader.filterMap = parameters.filterMap
-        config.photoRefObjLoader.filterMap = parameters.filterMap
+  calibrateImage:
+    class: lsst.pipe.tasks.calibrateImage.CalibrateImageTask
```

Comment:
Those connections can safely change?

Reply:
Yes. It's unfortunate that we're changing refcats here in the pipelines at all, cause in most other cases we do it in obs_ configs and we recently had some agreement on Slack that obs_ configs were preferred.  But I didn't want that scope on this ticket.